services/api-server/tests/unit/test_services_directorv2.py

```diff
@@ -0,0 +1,71 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+import pytest
+from fastapi import FastAPI, status
+from httpx import AsyncClient
+from models_library.projects import ProjectID
+from models_library.users import UserID
+from respx import MockRouter
+from settings_library.director_v2 import DirectorV2Settings
+from simcore_service_api_server.exceptions.backend_errors import JobNotFoundError
+from simcore_service_api_server.services.director_v2 import DirectorV2Api
+
+
+@pytest.fixture
+def api() -> DirectorV2Api:
+    settings = DirectorV2Settings()
+    app = FastAPI()
+
+    return DirectorV2Api.create_once(
+        app=app,
+        client=AsyncClient(base_url=settings.base_url),
+        service_name="director_v2",
+    )
+
+
+async def test_oec_139646582688800_missing_ctx_values_for_msg_template(
+    mocked_directorv2_service_api_base: MockRouter,
+    project_id: ProjectID,
+    user_id: UserID,
+    api: DirectorV2Api,
+):
+    #
+    # tests reported OEC:139646582688800  (SEE https://monitoring.tip.science/graylog/messages/graylog_33/a2444501-597a-11ef-a116-0242c0a80111)
```

Comment:
Note: We do not keep graylog data for a long time, so I think adding here the link or OEC ID doesn't help much.

Reply:
I know. It is more as reference. I copied the important parts in the test